lib/breakfast.js

```diff
@@ -0,0 +1,53 @@
+"use strict"
+
+var global = require('./global-submit.js')
+
+class Breakfast {
+  constructor(food=0, calories=0){
+    this.food = food;
+    this.calories = calories;
+  }
+
+storeBreakfast() {
+    var breakfastJSON = localStorage.getItem('breakfast')
+    if(breakfastJSON === null){
+      breakfastJSON = '[]';
+    };
+
+    var currentBreakfast = JSON.parse(breakfastJSON);
+    currentBreakfast.unshift({food: this.food, calories: this.calories});
+    var breakfastJSON = JSON.stringify(currentBreakfast);
+    localStorage.setItem('breakfast', breakfastJSON)
+}
+
+deleteBreakfast(event, that) {
+    event.preventDefault();
+    var breakfast = JSON.parse(localStorage["breakfast"]);
+    for (var i = 0; i < breakfast.length; i++) {
+      if ($(that).parent().parent().data('id') === i){
+        that.parentElement.parentElement.remove();
+        breakfast.splice(i,1)
+        var breakfastJSON = JSON.stringify(breakfast);
+        localStorage.setItem('breakfast', breakfastJSON);
+      }
+     }
+   };
+
+updateBreakfast(storage, tableName){
```

Comment:
We may decide to move delete method into its own global-delete file and pass in arguments like you did with submit. Alternatively, we can put the function back into index.js and pass arguments. Either way, the code gets dried up quite a bit.